doc/proposal/component-descriptor-v2-schema.yaml

```diff
@@ -110,6 +153,10 @@ definitions:
         type: 'array'
         items:
           $ref: '#/definitions/label'
+      digest:
+        oneOf:
+          - type: 'null'
+          - $ref: '#/definitions/digestSpec'
```

Comment:
According to the documentation there is no digest for sources. I do not understand why and would prefer to have also digests for sources but we need to clarify this with Enrico.